pkg/storage/etcd/etcd_test_util.go

```diff
@@ -155,3 +160,32 @@ func NewEtcdTestClientServer(t *testing.T) *EtcdTestServer {
 	}
 	return server
 }
+
+// NewEtcdTestStorage creates a new storage.Interface and TestServer
+func NewEtcdTestStorage(t *testing.T, codec runtime.Codec, prefix string) (*EtcdTestServer, storage.Interface) {
+	server := NewEtcdTestClientServer(t)
+	storage := NewEtcdStorage(server.client, codec, prefix)
+	return server, storage
+}
+
+// CreateObj will create a single object using the storage interface
+func CreateObj(t *testing.T, helper storage.Interface, name string, obj, out runtime.Object, ttl uint64) error {
+	err := helper.Set(context.TODO(), name, obj, out, ttl)
+	if err != nil {
+		t.Errorf("Unexpected error %v", err)
+	}
+	return err
+}
+
+// CreatePodList will properly create a podlist using the storage interface
+func CreatePodList(t *testing.T, prefix string, helper storage.Interface, list *api.PodList) error {
```

Comment:
I think runtime package has generic extract list items function. I think you should be able to use that without specifically referencing PodList. I agree with @wojtek-t-- I don't want to expand the import list unless it's unavoidable.